### ISSUE TITLE
Do not loot the Missives board if mod is installed

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#126]
Special case blacklist. QUST placed objects are already handled OK. This check has to be patch-aware. The Name of the container always seems to be Missive Board, in base mod and patches.

**Comment by SteveTownsend** [comment]
Test evidence from local Load Order
```
J:\GitHub\SmartHarvestSE\src\Data\dataCase.cpp(482): [MESSAGE] {35144} (2020-07-14 07:21:21 AM) Ignoring non-Vendor Container Missive Board/0xcb0012cb
J:\GitHub\SmartHarvestSE\src\Data\dataCase.cpp(482): [MESSAGE] {35144} (2020-07-14 07:21:21 AM) Ignoring non-Vendor Container Missive Board/0xcc0008da
J:\GitHub\SmartHarvestSE\src\Data\dataCase.cpp(482): [MESSAGE] {35144} (2020-07-14 07:21:21 AM) Ignoring non-Vendor Container Missive Board/0xcc0008ef
J:\GitHub\SmartHarvestSE\src\Data\dataCase.cpp(482): [MESSAGE] {35144} (2020-07-14 07:21:21 AM) Ignoring non-Vendor Container Missive Board/0xcc000a61
```

**Comment by SteveTownsend** [comment]
Fixed for v1.8.0.0
